packages/core/src/web/helpers/fonts/googleFontService.ts

```diff
@@ -0,0 +1,259 @@
+import { match, P } from 'ts-pattern';
+
+import { useStorageStore } from '@core/app/stores/storageStore';
+import localFontHelper from '@core/implementations/localFontHelper';
+
+import { useGoogleFontStore } from '../../app/stores/googleFontStore';
+
+import { generateGoogleFontPostScriptName, generateStyleFromWeightAndItalic } from './fontUtils';
+import googleFonts from './webFonts.google';
+
+/**
+ * Load static Google Fonts (predefined web fonts) with CSS only
+ */
+const loadStaticGoogleFonts = (lang: string): void => {
+  const googleLangFonts = googleFonts.getAvailableFonts(lang);
+
+  // Apply CSS styles (existing static font loading)
+  googleFonts.applyStyle(googleLangFonts);
+
+  // Track static fonts as loaded in the store
+  googleLangFonts.forEach(({ family }) => {
+    if (family) {
+      // Directly update store state for static fonts (bypass filtering)
+      const state = useGoogleFontStore.getState();
+
+      useGoogleFontStore.setState({ sessionLoadedFonts: new Set(state.sessionLoadedFonts).add(family) });
+    }
+  });
+};
+
+/**
+ * Normalize font family names for robust comparison
+ * Handles various font name formats and edge cases
+ */
+const normalizeFontName = (fontName: string): string =>
+  fontName
+    .toLowerCase()
+    .trim()
+    .replace(/^['"]+|['"]+$/g, '') // Remove surrounding quotes
+    .replace(/\s+/g, ' ') // Normalize whitespace
+    .replace(/[^\w\s-]/g, '') // Remove special characters except spaces and hyphens
+    .trim();
+
+/**
+ * Enhanced local font detection using localFontHelper
+ * This leverages the same sophisticated font detection logic that fontHelper uses
+ */
+const isLocalFont = (fontFamily: string): boolean => {
+  const foundFont = localFontHelper.findFont({ family: fontFamily });
+
+  if (foundFont && foundFont.family === fontFamily) {
+    return true;
+  }
+
+  // Additional check: get all available local fonts and do normalized comparison
+  const localFonts = localFontHelper.getAvailableFonts();
+  const normalizedTarget = normalizeFontName(fontFamily);
+  const directMatch = localFonts.some((font) => normalizeFontName(font.family) === normalizedTarget);
+
+  if (directMatch) {
+    return true;
+  }
+
+  return false;
+};
```

Comment:
This `isLocalFont` and another `isLocalFont` in `googleFontStore/utils/detection.ts` confuse me a little bit.
Are they designed differently on purpose?